deploy/olm-catalog/cluster-network-addons/0.0.1/cluster-network-addons-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,289 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: kubevirt-cluster-network-addons-operator.v0.0.1
+  namespace: placeholder
+  annotations:
+    capabilities: "Full Lifecycle"
+    categories: "Virtualization"
+    alm-examples: |
+      [
+        {
+          "apiVersion":"networkaddonsoperator.network.kubevirt.io/v1alpha1",
+          "kind":"NetworkAddonsConfig",
+          "metadata": {
+            "name":"cluster",
+          },
+          "spec": {
+            "multus":{},
+            "linuxBridge":{},
+            "sriov":{},
+            "kubeMacPool": {
+              "startPoolRange": "02:00:00:00:00:00",
+              "endPoolRange": "FD:FF:FF:FF:FF:FF",
+            }
+            "imagePullPolicy": "Always",
+          }
+        }
+      ]
+    description: Additional networking components for Kubernetes
+spec:
+  displayName: KubeVirt Cluster Network Addons
```

Comment:
There are no KubeVirt specific addons. The name is mentioned elsewhere too. For maintainers it makes sense. I don't want to nitpick too much, I can change it later. I just want to understand the reasoning behind it.

Reply:
Will update.